src/components/common/Title.tsx

```diff
@@ -1,25 +1,46 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import { COLOR } from '../../constant/color';
 
 const S = {
   Title: styled.h2`
-    font-weight: 500;
-    font-size: 20px;
-    line-height: 22px;
     display: flex;
     align-items: center;
-    color: #383838;
     cursor: pointer;
   `,
+  Text: styled.span`
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 1;
+    color: ${COLOR.BLACK};
+  `,
+  SvgWrapper: styled.div<{ isArrow: boolean }>`
```

Comment:
```suggestion
  SvgWrapper: styled.div<ITitle['isArrow']>`
```

이런식으로도 할 수 있겠네요!
두 타입이 만약 같은 값을 바라본다면, 관리포인트를 두 개로 하는 것 보다는 하나로 줄이는 게 나으니까요.

Reply:
같은 타입을 바라보고 있으니 하나로 관리해주는게 훨씬 좋네요!